tests/test_freqtradebot.py

```diff
@@ -1128,6 +1128,9 @@ def test_handle_stoploss_on_exchange(mocker, default_conf_usdt, fee, caplog, is_
     assert trade.is_open is False
     caplog.clear()
 
+    # TODO-lev: Test 2 identical orders but one with leverage, 1 without, and test that the
+    # leveraged trade is hit, but the other trade is not
```

Comment:
Why are we expecting the stoploss to be hit in the code a few lines above this?

Reply:
it's a stoploss on exchange - so the exchange returns "status=closed"  when asking for the order.

Price didn't have to move - and i'm not sure we must adapt this part for leverage either, as the exchange will still raise the stop when necessary (what we should test is that the value we place it is properly calculated)

Reply:
I was just trying to test [these changes](https://github.com/freqtrade/freqtrade/pull/6466/files#r815318556), but given what you said I don't think that would be tested here

Reply:
there's a difference - the changes are fine and will change how/where the stoploss will be placed.
That'll not change the "stop on exchange" handling though